src/PrestaShopBundle/Form/Admin/Sell/Supplier/SupplierType.php

```diff
@@ -110,40 +115,32 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('name', TextType::class, [
                 'constraints' => [
                     new NotBlank([
-                        'message' => $this->translator->trans(
-                            'This field cannot be empty', [], 'Admin.Notifications.Error'
+                        'message' => $this->trans(
+                            'This field cannot be empty', 'Admin.Notifications.Error'
                         ),
                     ]),
                     new Length([
                         'max' => SupplierSettings::MAX_NAME_LENGTH,
-                        'maxMessage' => $this->translator->trans(
+                        'maxMessage' => $this->trans(
                             'This field cannot be longer than %limit% characters',
-                            ['%limit%' => SupplierSettings::MAX_NAME_LENGTH],
-                            'Admin.Notifications.Error'
+                            'Admin.Notifications.Error',
+                            ['%limit%' => SupplierSettings::MAX_NAME_LENGTH]
                         ),
                     ]),
                     new TypedRegex([
                         'type' => TypedRegex::TYPE_CATALOG_NAME,
                     ]),
                 ],
             ])
-            ->add('description', TranslatableType::class, [
-                'type' => TextareaType::class,
-                'required' => false,
+            ->add('description', TranslateType::class, [
+                'type' => FormattedTextareaType::class,
+                'locales' => $this->locales,
+                'hideTabs' => false,
                 'options' => [
                     'constraints' => [
                         new CleanHtml([
-                            'message' => $this->translator->trans(
+                            'message' => $this->trans(
                                 '%s is invalid.',
-                                [],
-                                'Admin.Notifications.Error'
-                            ),
-                        ]),
-                        new TypedRegex([
```

Comment:
I removed this strange validation that - it seems - has nothing to do with a description field 🤔